pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -429,10 +438,10 @@ func (r *ReconcileGitTrack) updateChild(foundGTO, childGTO farosv1alpha1.GitTrac
 // deleteResources deletes any resources that are present in the given map
 func (r *ReconcileGitTrack) deleteResources(leftovers map[string]farosv1alpha1.GitTrackObjectInterface) error {
 	for name, obj := range leftovers {
-		log.Printf("Deleting child '%s'\n", name)
 		if err := r.Delete(context.TODO(), obj); err != nil {
 			return fmt.Errorf("failed to delete child for '%s': '%s'", name, err)
 		}
+		r.log.V(2).Info("Deleted child", "ChildName", name)
```

Comment:
Nitpick: you log "child created" and "child updated", but "deleted child" - maybe we could pick either order for all?

Reply:
I had deliberately chosen `Child <verb>` but clearly fell asleep doing that last one, good spot!